src/vcpkg/registries.cpp

```diff
@@ -873,6 +873,7 @@ namespace
         constexpr static StringLiteral BASELINE = "baseline";
         constexpr static StringLiteral PATH = "path";
         constexpr static StringLiteral REPO = "repository";
+        constexpr static StringLiteral REFERENCE = "reference";
```

Comment:
We might want to bikeshed the user-visible name of this field; I feel like there might be something more descriptive of the purpose (`"head"`? `"updates"`?)